docs/how-to-guides/azure_resource_provision.json

```diff
@@ -18,57 +18,45 @@
     },
     "allowAllConnections": {
       "defaultValue": "true",
-      "allowedValues": [ "true", "false" ],
+      "allowedValues": [
+        "true",
+        "false"
+      ],
       "type": "String",
       "metadata": {
         "description": "Specifies whether to allow client IPs to connect to Synapse"
       }
     },
     "provisionPurview": {
       "defaultValue": "true",
-      "allowedValues": [ "true", "false" ],
+      "allowedValues": [
+        "true",
+        "false"
+      ],
       "type": "String",
       "metadata": {
         "description": "Whether or not put purview in the provision script"
       }
     },
     "provisionEventHub": {
       "defaultValue": "true",
-      "allowedValues": [ "true", "false" ],
+      "allowedValues": [
+        "true",
+        "false"
+      ],
       "type": "String",
       "metadata": {
         "description": "Whether or not to deploy eventhub provision script"
       }
     },
-    "databaseServerName": {
-      "type": "string",
-      "defaultValue": "[concat('server-', uniqueString(resourceGroup().id, deployment().name))]",
-      "metadata": {
-        "description": "Specifies the name for the SQL server"
-      }
-    },
-    "databaseName": {
-      "type": "string",
-      "defaultValue": "[concat('db-', uniqueString(resourceGroup().id, deployment().name), '-1')]",
-      "metadata": {
-        "description": "Specifies the name for the SQL database under the SQL server"
-      }
-    },
-    "location": {
-      "type": "string",
-      "defaultValue": "[resourceGroup().location]",
-      "metadata": {
-        "description": "Specifies the location for server and database"
-      }
-    },
-    "adminUser": {
-      "type": "string",
+    "sqlAdminUsername": {
+      "type": "String",
       "metadata": {
         "description": "Specifies the username for admin"
       }
     },
-    "adminPassword": {
-      "type": "securestring",
+    "sqlAdminPassword": {
+      "type": "SecureString",
       "metadata": {
         "description": "Specifies the password for admin"
```

Comment:
>Specifies the password for admin [](http://example.com/codeflow?start=24&length=32)

Enya tried a deployment and she thought this one is for setup admin username and password for feathr. Can we make description more meaningful so user know this is for sql admin?

Reply:
I will make it more clear in upcoming PRs.